src/components/custom/_signup-new.js

```diff
@@ -108,6 +108,11 @@ const SocialButton = styled(Button)`
         width: 20rem;
         height: 6rem;
     }
+    @media ${device.mobileS} {
```

Comment:
currently the responsive orientation for deriv.com is from big to small, mobileS, is the smallest, on mobileM, the issue will still persist, instead, use mobileM or mobileL breakpoints to do this will resolve the problem imo